keyboards/rart/rart75hs/rart75hs.c

```diff
@@ -12,20 +12,4 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-#include "rart75hs.h"
```

Comment:
This should be left.  It's useful in the case that somebody is compiling from the configurator.

Reply:
ah i see, Allrights @drashna :)